test-runtimes/nextjs/src/pages/api/edge.ts

```diff
@@ -0,0 +1,28 @@
+import { NextResponse } from "next/server";
+
+import * as stytch from "stytch";
+
+const client = new stytch.Client({
+  // Find these values at https://stytch.com/dashboard/api-keys
+  // These ones are MADE UP!
+  project_id: "project-live-c60c0abe-c25a-4472-a9ed-320c6667d317",
+  secret: "secret-live-80JASucyk7z_G8Z-7dVwZVGXL5NT_qGAQ2I=",
```

Comment:
```suggestion
  project_id: "YOUR_STYTCH_PROJECT_ID",
  secret: "YOUR_STYTCH_SECRET",
```

This is the pattern we use elsewhere, dealer's choice on which you'd prefer.

Reply:
The secrets I'm checking in here are a special set we used to have public on the homepage - they produce a specific error and make it easy to confirm network connectivity. I'll add better comments.